### PR TITLE
Develop/jcsz1 fusy 13

### PR DESCRIPTION
**Pull request by djagiellowicz**

### CODE REVIEW

File: WalutyMVCWebApp/Views/_ViewStart.cshtml [hunk 1]
```diff
@@ -0,0 +1,3 @@
+﻿@{
+    Layout = "_Layout";
```
Comment (TDerkacz) [inline WalutyMVCWebApp/Views/_ViewStart.cshtml]: If we state "_Layout" will be good to have this file in: \Views\Shared\_Layout.cshtml

File: WalutyMVCWebApp/wwwroot/css/site.css [hunk 2]
```diff
@@ -0,0 +1,56 @@
+﻿/* Please see documentation at https://docs.microsoft.com/aspnet/core/client-side/bundling-and-minification
+for details on configuring this project to bundle and minify static web assets. */
+
```
Comment (TDerkacz) [inline WalutyMVCWebApp/wwwroot/css/site.css]: We cannot be surprised that css is uploaded with classes

File: WalutyMVCWebApp/Views/_ViewStart.cshtml (hunk 1 above)
Comment (djagiellowicz) [reply WalutyMVCWebApp/Views/_ViewStart.cshtml]: https://user-images.githubusercontent.com/36965101/54219453-d4d34980-44ef-11e9-83db-ff13cf0ef1d3.png there is

File: WalutyMVCWebApp/wwwroot/css/site.css (hunk 2 above)
Comment (djagiellowicz) [reply WalutyMVCWebApp/wwwroot/css/site.css]: From template

File: WalutyMVCWebApp/Views/_ViewStart.cshtml (hunk 1 above)
Comment (TDerkacz) [reply WalutyMVCWebApp/Views/_ViewStart.cshtml]: Sorry, my bad :) probably I wasn't looking in right folder

Comment (djagiellowicz) [reply WalutyMVCWebApp/Views/_ViewStart.cshtml]: :)

**Pull request merged**